website/src/styles/_variables.scss

```diff
@@ -0,0 +1,13 @@
+$primary: #00ADE6;
+$secondary: #05E3D5;
+$black: #041725;
+$black-20: #213746;
+$black-40: #5A666D;
+$white: #EDEDED;
+$white-40: #8D9BA3;
+
+$footer-height: 13rem;
+$topbar-height: 4rem;
+$topbar-maxheight: 12rem;
+$toc-width: 17rem;
+$mobile: 576px;
```

Comment:
I think it would be better if components were centered on the page? Like, 

![screen shot 2017-09-07 at 10 57 07 am](https://user-images.githubusercontent.com/6854312/30177643-55d887bc-93bb-11e7-9baa-ddd0e015aaca.png)

Looks weird to me